torch_ort_inference/torch_ort_inference/ortinferencemodule/utils.py

```diff
@@ -0,0 +1,60 @@
+import copy
```

Comment:
Please remove any unnecessary imports from the file.

Reply:
Noted. PR updated.